Sources/Rendering/Core/RenderWindowInteractor/index.js

```diff
@@ -1108,6 +1114,10 @@ function vtkRenderWindowInteractor(publicAPI, model) {
     superDelete();
   };
 
+  publicAPI.setMouseScrollDebounceBypass = (byPass) => {
+    model.bypassDebounce = byPass;
+  };
```

Comment:
@jourdain - if I understand what you're asking for https://github.com/Kitware/vtk-js/pull/2723#issuecomment-1441073973 correctly, you're asking if I can remove this and do something like
```
vtkRenderWindowInteractor.extend('vtkRenderWindowInteractor', 'bypassDebounce', true);
```
from the `RemoteView` function?

Reply:
No sorry, I meant in `Sources/Rendering/Core/RenderWindowInteractor/index.js` why you are not using the (set) macro that will trigger a `modified()` when `setMouseScrollDebounceBypass` get called. (That means you do not need to write that function. 

Then for the other classes, can we just leave it as-is and rely on the fact that you get the interactor from getters and set it on the interactor only?

Reply:
Basically just doing the following change (+updating the TS).

__Sources/Rendering/Core/RenderWindowInteractor/index.js__

```javascript

const DEFAULT_VALUES = {
	// ...
	mouseScrollDebounceByPass: false,
}

export function extend(publicAPI, model, initialValues = {}) {
	// ...
	macro.setget(publicAPI, model, [
            'mouseScrollDebounceByPass',
        ]);
	// ...
}

```

Reply:
And that part that I forgot

```js
    if (model. mouseScrollDebounceByPass) {
      publicAPI.extendAnimation(600);
      publicAPI.endMouseWheelEvent();
      model.wheelTimeoutID = 0;
    } else {
      // start a timer to keep us animating while we get wheel events
      model.wheelTimeoutID = setTimeout(() => {
        publicAPI.extendAnimation(600);
        publicAPI.endMouseWheelEvent();
        model.wheelTimeoutID = 0;
      }, 200);
    }
```

Reply:
will implement. Thank you!